.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         type: string
     working_directory: /opt/app-root/zync
     docker:
-      - image: registry.access.redhat.com/ubi7/ruby-27
+      - image: quay.io/3scale/zync:test-base
```

Comment:
The problem with the `registry.access.redhat.com/ubi9/ruby-31` image is that it lacks PostgreSQL libraries, and I didn't manage to install them with `dnf` as a CircleCI run step.

So, **temporarily** (just to check what other issues I may encounter) I replaced the ubi image with a custom one, built from the following Dockerfile:
```
FROM registry.access.redhat.com/ubi9/ruby-31

USER root
RUN rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
    && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs --skip-broken -y shared-mime-info postgresql13 postgresql13-libs \
    && dnf clean all \
    && rm -rf /var/cache/yum

USER default
```